ParseLiveQuery/src/main/java/com/parse/ParseLiveQueryClientImpl.java

```diff
@@ -32,7 +33,11 @@
     private boolean userInitiatedDisconnect = false;
 
     /* package */ ParseLiveQueryClientImpl(URI uri) {
-        this(uri, new TubeSockWebSocketClient.TubeWebSocketClientFactory(), Task.BACKGROUND_EXECUTOR);
+        this(uri, new OkHttp3SocketClientFactory(new OkHttpClient()), Task.BACKGROUND_EXECUTOR);
+    }
+
+    /* package */ ParseLiveQueryClientImpl(URI uri, OkHttpClient okHttpClient) {
+        this(uri, new OkHttp3SocketClientFactory(okHttpClient), Task.BACKGROUND_EXECUTOR);
```

Comment:
Good idea.  Changed.  

I kept the default implementation (line 36) as the default.